watchlist/src/utils/address.ts

```diff
@@ -0,0 +1,14 @@
+import { decodeAddress, encodeAddress } from '@polkadot/util-crypto'
+
+export function isValidAddress(address: string) {
+  try {
+    decodeAddress(address)
+    return true
+  } catch (e) {
+    return false
+  }
+}
```

Comment:
try is adress valid from the same package ;))

Reply:
Changed to use [isAddress](https://github.com/polkadot-js/common/blob/095db555ba6341ffd83736fe7285eb90608f34f1/packages/util-crypto/src/address/is.ts#L8C17-L8C26)